common/locals.tf

```
@@ -0,0 +1,42 @@
+locals {
+  heartbeat_filters_gcp   = "(not filter('gcp_status', '{Code=3, Name=STOPPING}', '{Code=4, Name=TERMINATED}'))"
```

```suggestion
  not_running_vm_filters_gcp   = "(not filter('gcp_status', '{Code=3, Name=STOPPING}', '{Code=4, Name=TERMINATED}'))"
```
This filter may be used for other things than heartbeat.